coins/zer.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@egyptianbman whats the issue? I open it locally it looks exactly like the other ones. No space is there if thats what it looks like.

Let me know. Thanks!

Reply:
It looks like some of the lines are using tabs instead of spaces, leading to weird alignment: 
![image](https://user-images.githubusercontent.com/323600/43848006-a04a3284-9af7-11e8-8e1e-2b035526c816.png)

Reply:
@egyptianbman look better?